src/app/user-page/user-page.component.html

```diff
@@ -13,6 +13,12 @@
       <img mat-card-image [src]="user?.avatarUrl" class="mt-0 mx-auto mb-3" alt="Dockstore avatar" />
       <mat-card-content fxLayout="column" fxLayoutAlign="center center">
         <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
+        <mat-card-subtitle *ngIf="gitHubProfile.bio" class="normal-line-height weight-bold accent-1-dark mt-2 mb-0 text-center-align">
+          {{ gitHubProfile.bio }}
+        </mat-card-subtitle>
+        <a rel="noopener noreferrer" href="{{ gitHubProfile?.link }}" target="_blank" class="accent-1-dark">
```

Comment:
Right now, it doesn't show a link if you don't have one because it's just an empty `<a>` tag. Can add `*ngIf="gitHubProfile?.link` to only add the `<a>` tag if the link exists